tests/Unit/Core/InputValidatorTest.php

```diff
@@ -1197,4 +1197,43 @@ public function testGetCompanyVatInValidator_DefaultTurnedOffOnline()
         $this->assertSame(1, count($aCheckers));
         $this->assertFalse($aCheckers[0] instanceof \OxidEsales\EshopCommunity\Core\OnlineVatIdCheck);
     }
+
+    public function providerEmailsToValidate()
+    {
+        return [
+            ['mathias.krieck@oxid-esales.com', true],
+            ['mytest@com.org', true],
+            ['my+test@com.org', true],
+            ['mytest@oxid-esales.museum', true],
+            ['?mathias.krieck@oxid-esales.com', true],
+            ['my/test@com.org', true],
+            ['mytest@-com.org', false],
+            ['@com.org', false],
+            ['mytestcom.org', false],
+            ['foo.bar@-.-,-,-.oxid-esales.com', false],
+            ['mytest@com', false],
+            ['info@�vyturys.lt', false],
+        ];
```

Comment:
You could add `abc@def` to the list, it should pass now, but I guess that's not an expected mail address.

Reply:
Hey @alfredbez,
with `mytest@com` we have a email address with a similar schem in the data provider which evaluates to `false` -> not a valid email address. In my personal opinion `abc@def` should also be not accepted.
/Flo

Reply:
oh, I missed that :man_facepalming: